agent/src/agent/pipeline/json_builder/json_builder.py

```diff
@@ -39,7 +38,8 @@ def _build_multiple(configs: list, build_func: Callable) -> List[Pipeline]:
             pipeline.manager.check_pipeline_id(config['pipeline_id'])
             pipelines.append(build_func(config))
         except Exception as e:
-            exceptions[config['pipeline_id']] = f'{type(e).__name__}: {traceback.format_exc()}'
+            exceptions[config['pipeline_id']] = f'{type(e).__name__}: {e}'
+            logger_.debug(e, exc_info=True)
```

Comment:
this is error so it should be `logger_.error(e, exc_info=True)`. Same for others